src/ssl.mli

```diff
@@ -204,6 +204,14 @@ val init : ?thread_safe:bool -> unit -> unit
     occurred. *)
 val get_error_string : unit -> string
 
+(** Retrieve the earliest error from the error queue then it removes the entry.
+    Returns the code and library and reason strings *)
+val get_error : unit -> int * string * string
+
+(** Retrieves the latest error code from the thread's error queue without
+    modifying it. Returns the code and library and reason strings. *)
+val peek_error_last : unit -> int * string * string
```

Comment:
can we call it `peek_last_error` like the SSL call (https://www.openssl.org/docs/man3.1/man3/ERR_get_error.html)?

should we also provide a binding to `peek_error`?

Reply:
Not sure, you think it would be useful?